.nvmrc

```diff
@@ -0,0 +1 @@
+v16.20.1
```

Comment:
I would rather not add this tbh, you can see the versions of node we support building the library on in the gh workflows and adding config to enforce a specific patch version would just be annoying

Reply:
Yeah, I felt the same. I will remove this. I think we should add recommended `js` versions in `CONTRIBUTING.md`